cpanfile

```diff
@@ -30,6 +30,8 @@ requires 'CatalystX::RoleApplicator';
 requires 'CatalystX::Fastly::Role::Response', '0.06';
 requires 'CPAN::Repository::Perms';
 requires 'Config::ZOMG', '>=', '1.000000';
+requires 'Config::JFDI';
```

Comment:
I *think* we should no longer be using `Config::JFDI` as of an earlier merge, so this can be removed.